test/python/test_rticonnextdds_input.py

```diff
@@ -49,3 +49,14 @@ def test_creation_DR(self,rtiInputFixture):
       and isinstance(rtiInputFixture.connector,rti.Connector) \
       and isinstance(rtiInputFixture.samples,rti.Samples) \
       and isinstance(rtiInputFixture.infos,rti.Infos)
+
+  def test_reader_native_call(self, rtiInputFixture):
```

Comment:
rtI_input_fixture?

Reply:
I'm not changing pre-existing test code at this moment.